schemas/mapping_schema.json

```diff
@@ -44,7 +44,7 @@
                 },
                 "authors": {
                     "type": "array",
-                    "description": "Lists the author(s) of the mapping table as a name, email, or company.",
+                    "description": "Lists the author(s) of the mapping table as a name, email, or company. Multiple authors should be comma-separated.",
```

Comment:
I would remove the sentence about comma-separation, because it is not really true. The data type is an array, so there is no need for manual comma-separation.

Reply:
Removed sentences.